src/main/java/io/github/moderngamingworlds_mods/woodenutilities/common/recipes/WoodcutterRecipe.java

```diff
@@ -0,0 +1,147 @@
+package io.github.moderngamingworlds_mods.woodenutilities.common.recipes;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.github.moderngamingworlds_mods.woodenutilities.WoodenUtilities;
+import io.github.moderngamingworlds_mods.woodenutilities.common.init.ModBlocks;
+import io.github.moderngamingworlds_mods.woodenutilities.common.init.ModRecipes;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.world.Container;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraft.world.item.crafting.Recipe;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.ForgeRegistryEntry;
+import org.jetbrains.annotations.Nullable;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class WoodcutterRecipe implements Recipe<Container> {
+
+    public static final String ID = WoodenUtilities.getInstance().modid.concat("_woodcutter");
+    public static final String SERIALIZER_ID = "woodcutter_serializer";
+
+    public ResourceLocation recipeLoc;
+
+    public final String group = WoodenUtilities.getInstance().modid.concat(":").concat(SERIALIZER_ID);
```

Comment:
`group` is never used. Intended?

Reply:
Noup. The group isnt intended to change but I forgot to remove it
Done in b3a58f9